src/components/CornerDialog/CornerDialog.props.ts

```diff
@@ -0,0 +1,13 @@
+import type { ReactNode } from 'react';
+
+import { CornerDialogConfig } from './CornerDialog.styles';
+import { DefaultButtonProps } from '../Button/Button.props';
+
+export type CornerDialogProps = {
+  custom?: CornerDialogConfig;
+  intent?: 'none' | 'warning' | 'negative';
+  title: string;
+  content: ReactNode;
+  actions?: DefaultButtonProps[];
+  onCloseClick?: (e: React.MouseEvent) => void;
```

Comment:
```suggestion
import type { ReactNode, MouseEvent } from 'react';

  onCloseClick?: (e?: MouseEvent) => void;
```

`e` should be optional ;)